src/main/java/core/basesyntax/dao/StoreCsvDaoImpl.java

```diff
@@ -0,0 +1,149 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.entity.FruitTransaction;
+import core.basesyntax.entity.Operation;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.OutputStream;
+import java.io.OutputStreamWriter;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class StoreCsvDaoImpl implements StoreCsvDao {
```

Comment:
it's a weird class with a weird name TBH :)) What is CSV Dao? Deal with file reading, data processing, report generation, writing to a file separately